github/github.go

```diff
@@ -328,6 +328,9 @@ func (c *Client) GetOrCreatePR(owner, repo, head, base, title, body string, draf
 	if err != nil {
 		return nil, err
 	}
+	if pr != nil && pr.GetMerged() {
+		return pr, nil
+	}
```

Comment:
I think the root cause of kuboreleaser failing for RC2s was because it was finding PRs that were already merged. This should hopefully help with that.